<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>187</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting RotationMirror</string>
  </property>
  <widget class="QToolButton" name="toolButtonMirrorUpDown">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Upside Down</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonMirrorLeftRight">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Left-Right</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>RotationAngle</string>
   </property>
   <widget class="QRadioButton" name="radioButtonRot0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>86</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0 Degree</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonRot90">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>86</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>90 Degree</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonRot180">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>86</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>180 Degree</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonRot270">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>86</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>270 Degree</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
